### ISSUE TITLE
Dwarf origin health isn't correct

### ISSUE THREAD

**Issue by Jar420** [DistantDig/digs-dnd-origins#31]
I have 8 hearts as the Mountain Dwarf.

**Comment by DistantDig** [comment]
Yup! this is intentional. Dwarves got a buff in the most recent update so they have a greater overall health compared to other smaller races

**Comment by DistantDig** [comment]
im not sure if this was what you were meaning, but i see now that the description for the 1.19.2 version doesnt have the correct version
